operator/cilium_node.go

```diff
@@ -468,3 +489,39 @@ func updateCNP(ciliumClient v2.CiliumV2Interface, cnp *cilium_v2.CiliumNetworkPo
 		}
 	}
 }
+
+func ClearCnpStatusNodes(ctx context.Context) error {
+	clearStatusNodes := func(ctx context.Context, cnp *cilium_v2.CiliumNetworkPolicy) error {
+		data, err := json.Marshal([]k8s.JSONPatch{
+			{
+				OP:   "remove",
+				Path: "/status/nodes",
+			},
```

Comment:
Just to be sure about this, will the operation fail if the `/status/nodes` does not exist in the object or will be a no-op and no error is returned from kubernetes?

Reply:
It fails with the following reported error message: `the server rejected our request due to an error in our request`

Reply:
Ok, we should then ignore the error or check if the `/status/nodes` ir present with a `op: test` like [here](https://github.com/cilium/cilium/blob/314ca7baeff4f568ffc0bad95124e40665b1f88c/pkg/k8s/cnp.go#L348-L352)

Reply:
Reading the [JSON Patch RFC](https://datatracker.ietf.org/doc/html/rfc6902/#section-4.6) it seems that it is not possible to test just for the existence of a field regardless of its value.
Anyway, we can easily ignore the specific error type returned from the client.